src/main/blackbox/blackbox.c

```diff
@@ -1210,6 +1210,7 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE("Firmware date", "%s %s",                buildDate, buildTime);
         BLACKBOX_PRINT_HEADER_LINE("Log start datetime", "%s",              blackboxGetStartDateTime(buf));
         BLACKBOX_PRINT_HEADER_LINE("Craft name", "%s",                      pilotConfig()->name);
+        BLACKBOX_PRINT_HEADER_LINE("Pilot name", "%s",                      pilotConfig()->pilotName);
```

Comment:
There was some discussion recently about this feature, and it was agreed that there is no point in trying to be overly prescriptive and insinuating that this can only be used to store the pilot name. It was agreed that the naming to go for was `Display name`, to indicate that this was to be used for display in the OSD only, and not used for naming log files / configs, unlike 'craft name'.

This should all be renamed accordingly.